docs/nomad_simulations.md

```diff
@@ -1 +1,499 @@
+
+    What are the different formats of the chemical formula? What is the string of the `descriptive` format and why it coincides with other format(s) in the sub-section? 
+
+??? success "Solution 2.6"
+    We can import and create instances of `ModelSystem` and `AtomicCell` and assign the corresponding quantities directly: 
+    ```python
+    from nomad_simulations.schema_packages.model_system import ModelSystem, AtomicCell
+    model_system = ModelSystem()
+    atomic_cell = AtomicCell(atoms_state=atoms_states, positions=[[0, 0, 0], [1, 1, 1]] * ureg.meter)
+    model_system.cell.append(atomic_cell)
+    ```
+
+    We can append this section to `Simulation`:
+    ```python
+    simulation.model_system.append(model_system)
+    ``` 
+
+    We can now assign directly an empty `ChemicalFormula` section to `ModelSystem`, in order to be able to call for the specific class method:
+    ```python
+    from nomad_simulations.schema_packages.model_system import ChemicalFormula
+    model_system.chemical_formula = ChemicalFormula()
+    ```
+
+    In order to extract the formulas and assign them to the quantities of the section `ChemicalFormula`, we can use the method `normalize()`. This class method takes two arguments as inputs: `archive` and `logger`. For the purpose of this tutorial, we can set `archive=None`:
+    ```python
+    model_system.chemical_formula.normalize(archive=None, logger=logger)
+    ```
+
+    Now, the `ChemicalFormula` sub-section contains the information of the different formats for the chemical formula: `reduced` and `hill` are both `'AsGa'`, while `iupac` is `'GaAs'`. The `descriptive` formula is set depending on the chemical elements present in the system, and because neither `'H'` nor `'O'` is present (i.e., no organic formulas), and gallium arsenide does not belong to any format exceptions, then it is set to the inorganic typical formula `'GaAs'`, coinciding with `iupac`. The `anonymous` formula is `'AB'`.
+
+
+
+## `Outputs` {#outputs}
+
+The `Outputs` section is an output section which contains all the information about the computed properties by the simulation, as well as references to the specific `ModelMethod` and `ModelSystem` used to compute such properties. These properties are under `Outputs` in a long list of sub-sections, and they inherit from an abstract section called `PhysicalProperty`. This section contains the `value` of the property as well as other quantities. The `variables` sub-section allows to write a physical property in term of a varying parameter, and it also allows to calculate the shape of `value` in a dynamic way. This means that, the same physical property (e.g., `ElectronicBandGap`) could have different shapes depending on the problem being parsed (e.g., a single scalar number or a set of varying scalars with respect to a variable, e.g., `Temperature`).
```

Comment:
"The `Outputs` section contains all the information about the properties computed by the simulations, as well as references to the relevant `ModelMethod` and `ModelSystem`."

Reply:
"long list of sub-sections" is a little confusing cause you are talking about the list of attributes in a class not a repeating sub-section.

What about "Each property is stored individually under `Outputs`, and inherits from an abstract section call `PhysicalProperty`. The `PhysicalProperty` class contains the `value` of the property along with other relevant quantities. The variables sub-section enables the physical property to be stored as a function of a varying parameter. Accordingly, the shape of `value` is calculated in a dynamic way."

Reply:
"This means that the same physical property (e.g., `ElectronicBandGap`) could have different shapes depending on the use case..."